package.json

```diff
@@ -65,10 +65,12 @@
     "test": "npm run cover"
   },
   "dependencies": {
+    "@google-cloud/automl": "^0.1.2",
```

Comment:
I don't think we want to have vision depend on automl, do we?

Reply:
Fixed in latest commit.